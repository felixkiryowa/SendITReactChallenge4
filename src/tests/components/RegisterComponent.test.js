import React from 'react';
import renderer from 'react-test-renderer';
import RegisterComponent from '../../components/Auth/Register';

describe('Sample Articles Component', () => {
    let mountApp;
    const props = {
        handleChange: jest.fn(),
        handleSubmit:jest.fn(),
        errors: {

        },
    };
    beforeEach(() => {
        mountApp = renderer.create(<RegisterComponent {...props} />);
    });

    it('Should render without crashing', () => {
        expect(mountApp).toMatchSnapshot();
    });
});
