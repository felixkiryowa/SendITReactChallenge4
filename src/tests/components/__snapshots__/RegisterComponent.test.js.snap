// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Articles Component Should render without crashing 1`] = `
<section
  className="testimonial py-5"
  id="testimonial"
>
  <div
    className="container reg-container"
  >
    <div
      className="row "
    >
      <div
        className="col-md-4 py-5 custom-card-color text-white text-center "
      >
        <div
          className=" "
        >
          <div
            className="card-body"
          >
            <h2
              className="py-3"
            >
              Registration
            </h2>
            <p>
              Create An Account With Send-IT such that your able to create parcel orders
            </p>
          </div>
        </div>
      </div>
      <div
        className="col-md-8 py-5 border"
      >
        <h4
          className="pb-4"
        >
          Please fill with your details
        </h4>
        <form
          onSubmit={[MockFunction]}
        >
          <div
            className="form-row"
          >
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                id="FirstName"
                name="firstname"
                onChange={[MockFunction]}
                placeholder="First Name"
                type="text"
              />
            </div>
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                name="username"
                onChange={[MockFunction]}
                placeholder="Username"
                type="text"
              />
            </div>
          </div>
          <div
            className="form-row"
          >
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                name="lastname"
                onChange={[MockFunction]}
                placeholder="Last Name"
                required="required"
                type="text"
              />
            </div>
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                name="password"
                onChange={[MockFunction]}
                placeholder="Password"
                required="required"
                type="password"
              />
            </div>
          </div>
          <div
            className="form-row"
          >
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                maxLength="10"
                name="contact"
                onChange={[MockFunction]}
                placeholder="Mobile No."
                required="required"
                type="text"
              />
            </div>
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                name="confirmPassword"
                onChange={[MockFunction]}
                placeholder="Confirm Password"
                required="required"
                type="password"
              />
            </div>
          </div>
          <div
            className="form-row"
          >
            <div
              className="form-group col-md-6"
            >
              <input
                className="form-control"
                name="useremail"
                onChange={[MockFunction]}
                placeholder="Email e.g daddyfelix56@gmail.com"
                required="required"
                type="email"
              />
            </div>
          </div>
          <br />
          <br />
          <div
            className="form-row"
          >
            <div
              className="form-group col-md-12"
            >
              <button
                className="btn  float-right reg-button"
                type="submit"
              >
                Sign Up
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</section>
`;
